tests/mir-opt/inline/inline_retag.rs

```diff
@@ -9,6 +8,8 @@ fn main() {
 
 // EMIT_MIR inline_retag.bar.Inline.after.mir
```

Comment:
```suggestion
// EMIT_MIR inline_retag.bar.Inline.diff
```
Making it a diff test is probably easier to monitor.